src/labels/CustomLabels.labels

```diff
@@ -2371,6 +2371,30 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>status on a bdi object when no matching record found</shortDescription>
         <value>Matched None</value>
     </labels>
+    <labels>
+        <fullName>bdiMatchedByUser</fullName>
+        <categories>Data Import</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>status on a BDI object when user has selected matching record in BGE interface</shortDescription>
```

Comment:
status -> Status

Reply:
(in multiple descriptions)

Reply:
Cool, fixed these in the places where we had copied Dave's code too.